diff --git a/lib/nitro-dev.js b/lib/nitro-dev.js
index 83b082422216bb63abf201550cc4b380db794e1b..1fe4826e21bbfd9b2d062fe3fcbf4460c2991ce5 100644
--- a/lib/nitro-dev.js
+++ b/lib/nitro-dev.js
@@ -293,26 +293,7 @@ export async function createDevServer(nitro) {
 	// 	}),
 	// );
 
-	const adapter = wsAdapter({
-		...wsApp.websocket,
-		hooks: {
-			open: (event) => {
-				event.ctx.node.req.url = event.ctx.node.req.originalUrl;
-			},
-			message: (event) => {
-				event.ctx.node.req.url = event.ctx.node.req.originalUrl;
-			},
-			close: (event) => {
-				event.ctx.node.req.url = event.ctx.node.req.originalUrl;
-			},
-			error: (event) => {
-				event.ctx.node.req.url = event.ctx.node.req.originalUrl;
-			},
-			upgrade: (event) => {
-				// event.ctx.node.req.url = event.ctx.node.req.originalUrl;
-			},
-		},
-	});
+	const adapter = wsAdapter(wsApp.websocket);
 	// Listen
 	/** @type {import("@vinxi/listhen").Listener[]}  */
 	let listeners = [];
